lib/checks/language/valid-lang.tpl

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Its probably not a good idea to require this task to be run every time; it would be better to run once every other release or so.  However, if this is the case we should get rid of the `.tpl` file and just:

1.  Read JSON file - `var check = grunt.file.readJSON(path)` [grunt.file.readJSON](http://gruntjs.com/api/grunt.file#grunt.file.readjson)
2.  Assign options -  `check.options = langs`
3.  Write JSON file - `grunt.file.write(path, JSON.stringify(check, null, '  '))` [grunt.file.write](http://gruntjs.com/api/grunt.file#grunt.file.write)

Grunt handles all the pathing issues for you, so its preferable to use grunt file methods where possible.

If we keep the .tpl file and JSON file in source control, my fear is they will quickly become out of sync.  Part of the reason these files are still JSON is this kind of portability.